psutil/_psutil_common.h

```diff
@@ -4,6 +4,9 @@
  * found in the LICENSE file.
  */
 
+#ifndef PSUTIL_PSUTIL_COMMON_H
+#define PSUTIL_PSUTIL_COMMON_H
```

Comment:
why is this necessary?

Reply:
This guards the header from being included multiple times into one translation unit (either directly/indirectly). In C, unless the header does something special, you always define such guards (or use the non-standardized once pragma).